subiquity/server/controllers/filesystem.py

```diff
@@ -179,6 +181,21 @@ def guided_lvm(self, disk, lvm_options=None):
                 mount="/",
                 ))
 
+    async def guided(self, choice):
```

Comment:
There's no reason to think this function will ever need to await is there?

Reply:
Can't come up with a reason, removing the async.